debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/SqlUtilsTest.java

```diff
@@ -321,11 +321,10 @@ public void shouldDetectConnectionProblems() {
     private String resolveLogMineryContentQueryFromTemplate(String schemaReplacement, String tableReplacement) {
         String query = LOG_MINER_CONTENT_QUERY_TEMPLATE;
 
-        List<String> excludedSchemas = OracleConnectorConfig.getExcludedSchemaNames();
-        if (!excludedSchemas.isEmpty()) {
+        if (OracleConnectorConfig.EXCLUDED_SCHEMAS.length > 0) {
             StringBuilder systemPredicate = new StringBuilder();
             systemPredicate.append("AND SEG_OWNER NOT IN (");
-            for (Iterator<String> i = excludedSchemas.iterator(); i.hasNext();) {
+            for (Iterator<String> i = Arrays.stream(OracleConnectorConfig.EXCLUDED_SCHEMAS).iterator(); i.hasNext();) {
```

Comment:
Tiny nit: couldn't `EXCLUDED_SCHEMAS` be an (unmodifiable) list, and you simply use a for each loop? Or is there any specific need for the iterator here?

Reply:
No particular reason, I can certainly change that.